pkg/machine/hyperv/machine.go

```diff
@@ -378,6 +387,9 @@ func getVMConfigPath(configDir, vmName string) string {
 func loadMacMachineFromJSON(fqConfigPath string, macMachine *HyperVMachine) error {
 	b, err := os.ReadFile(fqConfigPath)
 	if err != nil {
+		if os.IsNotExist(err) {
+			return machine.ErrNoSuchVM
```

Comment:
```suggestion
			return fmt.Errorf("%q: %w", fqConfigPath, machine.ErrNoSuchVM)
```
Not sure if the path is useful to users but in case it is.